adapter/service/outlvlswitch/service.go

```diff
@@ -50,6 +73,10 @@ type Service interface {
 	SetLevel(value int64, duration time.Duration) error
 	// SetBinaryState sets a binary value.
 	SetBinaryState(value bool) error
+	// StartLevelTransition starts a transition. Supported values are: "up" and "down"
+	StartLevelTransition(string, int, time.Duration) error
+	// StopLevelTransition stops a transition
+	StopLevelTransition() error
```

Comment:
Implement method "SupportLevelTransition".

Reply:
In the NewService constructor run method cfg.Specification.EnsureInterfaces with respective interfaces if the controller supports LevelTransitionController. Look at Chargepoint service for some references.